python/ray/serve/_private/api.py

```diff
@@ -94,17 +90,8 @@ def _start_controller(
     elif isinstance(global_logging_config, dict):
         global_logging_config = LoggingConfig(**global_logging_config)
 
-    controller = ServeController.options(
-        num_cpus=0,
-        name=SERVE_CONTROLLER_NAME,
-        lifetime="detached",
-        max_restarts=-1,
-        max_task_retries=-1,
-        resources={HEAD_NODE_RESOURCE_NAME: 0.001},
-        namespace=SERVE_NAMESPACE,
-        max_concurrency=CONTROLLER_MAX_CONCURRENCY,
-        enable_task_events=RAY_SERVE_ENABLE_TASK_EVENTS,
-    ).remote(
+    controller_handle = get_controller_handle()
+    controller = controller_handle.remote(
```

Comment:
```suggestion
    controller_impl = get_controller_impl()
    controller_handle = controller_impl.remote(
```

Either `get_controller_impl` or the original `get_controller` makes more sense. `ray.remote(ServeController)` only returns an actor definition (similar to `serve.deployment(ModelClass)`), and calling `x.remote()` will return the actual actor handle.